tools/defaults.bzl

```diff
@@ -79,6 +81,47 @@ def pkg_npm(name, use_prodmode_output = False, **kwargs):
         deps = deps,
     )
 
+    # Merge package.json with root package.json and perform various substitutions to
+    # prepare it for release.
+    jq(
+        name = "basic_substitutions",
+        srcs = ["//:package.json"] + kwargs.pop("srcs"),
+        filter = """
+            # Root package.json
+            .[0] as $root
+            # Project package.json
+            | .[1] as $proj
```

Comment:
Is it guaranteed that the project sub `package.json` is the second entry here? It seems like it could break when there is another JSON file at package top-level (like for migrations)

Reply:
You're right, this will break if the first thing passed to `srcs` in `pkg_npm` is not `package.json`.

Reply:
Updated to look through `srcs` and `deps` for package.json and explicitly use that.